docker-compose.yml

```diff
@@ -8,30 +8,33 @@ services:
     ports:
       - 6379
   postgres:
-    image: postgres
+    image: nickblah/postgis:11-bullseye-postgis-3
```

Comment:
Not sure about this. Do we need postgis@^2? I had to use `nickblah`'s image, because the official `postgis` docker image does not work on ARM (M1 Mac).

Reply:
yes postgis is necessary.

Reply:
i'm sure there is a way to add postgis on top through some command line.

also, isnt docker supposed to fix such issues? 😆 docker man...

Reply:
The image seems to work fine. My point is, that the image I am using brings postgis@3, not postgis@2. So the question is: Will the app work with postgis@3? I think today it uses postgis@2, right?

Reply:
🤷

Reply:
All the test pass, so I would say it should be okay. Maybe one wants to upgrade to postgis@3 also for deployment at some point. Feel free to resolve this conversation @phoet